README.md

```diff
@@ -21,6 +21,76 @@ The folder structure is as follows:
     - `cc-global-footer.vue`: The component for the Global CC footer.
     - `index.js`: Where our components are exported. Any new component must be registered in this file.
 
+## Components
+
+### CC Explore
+
+CC Global Footer requires a `donation-url` attribute which is the URL used for the Donation button.
+
+```html
+<cc-explore donation_url="http://example.com" />
```

Comment:
I think here we need to use the kebab case for the attribute?
`<cc-explore donation-url="http://example.com" />`